cni/network/network.go

```diff
@@ -1054,12 +1054,15 @@ func (plugin *NetPlugin) Delete(args *cniSkel.CmdArgs) error {
 				// Log the error but return success if the network is not found.
 				// if cni hits this, mostly state file would be missing and it can be reboot scenario where
 				// container runtime tries to delete and create pods which existed before reboot.
+				// this condition will not apply to stateless CNI since the network struct will be crated on each call
 				err = nil
-				return err
+				if !plugin.nm.IsStatelessCNIMode() {
```

Comment:
do we need this check if it doesn't apply to stateless cni?

Reply:
I added a comment on top of it :
// this condition will not apply to stateless CNI since the network struct will be crated on each call